payments/card/other-features.md

```diff
@@ -74,6 +74,58 @@ when="at the 3-D Secure verification for credit card payments" full_reference=tr
 
 {% include prices.md %}
 
+## 3-D Secure 2.0
+
+When dealing with card payments, 3-D Secure authentication of the
+cardholder is an essential topic. 3-D Secure 2.0 is an improved version of the
+old protocol, now allowing frictionless payments where transactions can be
+completed without input from the cardholder. Therefore, there are certain fields
+that should be included when implementing 3-D Secure 2.0. These are listed below
+and can be seen in the abbreviated request example below.
+
+*   `payment.riskIndicator`
+*   `payment.cardholder`
```

Comment:
This list can be removed when we have a full example, but let's add a field table below the example that highlights the 3-D Secure 2 related fields.